documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -121,6 +121,9 @@ Let's see the options for the `index_schema_management_strategy` property:
 |`MERGE`|Missing indexes and mappings will be created, existing mappings will be updated if there are no conflicts.
 |`CREATE`|**The default**: existing indexes will not be altered, missing indexes will be created along with their mappings.
 |`RECREATE`|Indexes will be deleted if existing and then created along with their mappings. This will delete all content from the indexes!
++
+Note that whenever a search factory is altered after initialization (i.e. new entities are mapped),
+the index will *not* be deleted again: new mappings will simply be added to the index.
```

Comment:
N.B. that for the typical Hibernate Search user there is no such thing as "adding new entities at runtime". This capability is an implementation detail of the Infinispan integration, and possibly of other integrators (beyond the Hibernate ORM integration).

This documentation is specific for the Hibernate Search / Hibernate ORM integration, so to not make it all too confusing for them I'd avoid such notes.

I'll edit this commit to remove this specific change, then merge it all.. I hope you don't mind?